global_costmap:
  global_frame: podi_map
  robot_base_frame: podi_base_link
  update_frequency: 5.0
  publish_frequency: 2.5
  plugins:
    - { name: static_layer, type: "costmap_2d::StaticLayer" }
    - { name: open_space_layer, type: "tbd_costmap::OpenSpaceLayer"}
    - { name: obstacle_layer, type: "costmap_2d::ObstacleLayer" }
    - { name: inflation_layer, type: "costmap_2d::InflationLayer" }

  open_space_layer:
    range_preferred: 0.6
    range_max: 0.8
    cost_weight: 0.95
